### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by MauricioUyaguari**
Fixes config to start dev server

### CODE REVIEW

File: packages/legend-studio-app/scripts/setup.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ fs.writeFileSync(
       appName: 'studio',
       env: 'local',
       sdlc: {
-        url: 'https://legend.finos.org/sdlc',
+        url: 'https://legend.finos.org/sdlc/api',
       },
       engine: {
         url: 'https://legend.finos.org/exec',
```
Comment (akphi) [inline packages/legend-studio-app/scripts/setup.js:49]: if that is the case for SDLC, why don't we have to do the same for EXEC?

Comment (MauricioUyaguari) [reply packages/legend-studio-app/scripts/setup.js:49]: always been the case :) we should change our clients to handle the /api in building the url. Right now all calls to legend sdlc required the /api prefix.

Comment (akphi) [reply packages/legend-studio-app/scripts/setup.js:49]: ok, let's not change the config setup here, let's change the client instead. Looks super weird from config perspective.

**Comment by akphi** [comment]
Closing in favor of https://github.com/finos/legend-studio/pull/59

**Pull request closed without merging**